src/domains/profile/pages/Welcome/Welcome.tsx

```diff
@@ -180,6 +180,29 @@ export const Welcome = () => {
 
 	const hasProfiles = profiles.length > 0;
 
+	useEffect(() => {
+		// The timeout prevents this action from running twice (apparently caused
+		// by lazy loading of this page). If removed, a toast with an error quickly
+		// appears and disappears.
+		let navigateTimeout: ReturnType<typeof setTimeout> | undefined;
+
+		if (!isDeeplink()) {
+			return;
+		}
+
+		navigateTimeout = setTimeout(() => {
+			if (profiles.length === 1) {
+				navigateToProfile(profiles[0]);
```

Comment:
Hey @alfonsobries I it should be also safe to mark it as clicked here, so it would ignore the user clicks on profile while validating URI.
```suggestion
				isProfileCardClickedOnce.current = true;
				navigateToProfile(profiles[0]);
```